hack/gcp-secret-manager/commands/list.py

```diff
@@ -0,0 +1,83 @@
+import json
+
+import click
+import requests
+import yaml
+from google.api_core.exceptions import PermissionDenied
+from google.cloud import secretmanager
+from util import CONFIG_PATH, PROJECT_ID, ensure_authentication, validate_collection
+
+
+@click.command("list")
```

Comment:
I would change the way secrets collection are presented when no collection has been specified by the user. Instead of a plain list:
```
collection_1
collection_2
...
collection_n
```
I would add the group information as well:
```
aos-qe:
- collection_1
- collection_2
art-admins:
- foo_collection
...
```